### PR TITLE
Fix website link and update source code versions.

### PR DESCRIPTION
**Pull request by GabrielBrascher**

### CODE REVIEW

**Comment by rafaelweingartner** [comment]
Could you squash the commits?

**Comment by GabrielBrascher** [comment]
@rafaelweingartner done.

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ The first user of this solution is the Network and Management Laboratory of the
  
 ## Getting source repository
 
-Autonomiccs Platform project uses Git and a mirror is hosted on <a href="https://github.com/Autonomiccs/autonomiccs-platform">GitHub</a>.
+Autonomiccs Platform project uses Git and a mirror is hosted on GitHub. To access our master source code, click <a href="https://github.com/Autonomiccs/autonomiccs-platform">here</a>. The latest stable version source code is with the tag <a href="https://github.com/Autonomiccs/autonomiccs-platform/tree/1.0.0">1.0.0</a>.
```
Comment (rafaelweingartner) [inline README.md]: what about changing the name of the link from 1.0.0 to only "stable", then we would need only to update the link for the tag when a new release is out.

Comment (GabrielBrascher) [inline README.md]: I thought about refer to the version explicitly, but no problem in changing it.

Comment (rafaelweingartner) [inline README.md]: me neither, it is just so we can discuss and reach a consensus. 
@pedro-martins would you mind joing us here?

Comment (rafaelweingartner) [inline README.md]: So, what about leaving it as it is, just improving the sentence.
from 

> The latest stable version source code is with the tag

to

> The latest stable version of Autonomiccs platform source code is: ...

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Mailing lists:
 
 Please, go to our <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki">wiki</a> page for more details; there you can understand more of the <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki/Autonomiccs-platform">Autonomiccs platform</a>, some <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki/Basic-concepts">basic concepts</a>, what are <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki/Heuristics">heuristics</a> and how we use it, how to <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki/Installation">install</a> and <a href="https://github.com/Autonomiccs/autonomiccs-platform/wiki/Usage">use</a>.
 
-You can find more about Autonomiccs visiting the <a href="autonomiccs.com.br">website</a>.
+You can find more about Autonomiccs visiting the [website](HTTP://autonomiccs.com.br).
```
Comment (rafaelweingartner) [inline README.md]: Shouldn't it be `http`, instead of `HTTP`

**Comment by GabrielBrascher** [comment]
@rafaelweingartner, I have updated.

**Comment by rafaelweingartner** [comment]
it seems better now.
I think we can merge this now.

**Pull request merged**